docs/get-started/install.rst

```diff
@@ -39,20 +44,50 @@ Do the following instead.
     brew install opencv
     brew info openblas
     brew install openblas
-    git clone --recursive https://github.com/dmlc/mxnet
+    git clone --recursive -b engine https://github.com/dmlc/mxnet
     cd mxnet; cp make/osx.mk ./config.mk; make -j$(sysctl -n hw.ncpu)
 
 It turns out that installing ``openblas`` is necessary, in addition to modify the makefile, to fix `one of the build issues <https://github.com/dmlc/mxnet/issues/572>`_.
 
 Setup Python and its environment
 --------------------------------
 
-Refer to MXNet installation document for `Python package installation. <http://mxnet.readthedocs.io/en/latest/how_to/build.html>`_. One of the most common problem a beginner runs into is not setting the environment variable to tell Python where to find the library. Suppose you have installed ``mxnet`` under your home directory and is running bash shell. Put the following line in your ``~/.bashrc`` (or ``~/.bash_profile``)
+Refer to MXNet installation document for `Python package installation. <http://mxnet.io/get_started/ubuntu_setup.html#install-mxnet-for-python>`_. One of the most common problem a beginner runs into is not setting the environment variable to tell Python where to find the library. Suppose you have installed ``mxnet`` under your home directory and is running bash shell. Put the following line in your ``~/.bashrc`` (or ``~/.bash_profile``)
 
 ::
 
     export PYTHONPATH=~/mxnet/python:$PYTHONPATH
```

Comment:
Yep, it will be installed in the $WORKON_HOME/minpy_dev. It's the same when you try to `pip install package`. What virtualenv does is changing the default package path.